test-framework/Vagrantfile

```diff
@@ -38,6 +39,26 @@ if ENV['GIT_BRANCH'].nil?
 end
 
 Vagrant.configure("2") do |config|
+  # Note: This requires Vagrant 2.2.0 or newer
+  config.trigger.before :up do |trigger|
+    trigger.info = "Acquiring Lock..."
+    trigger.ruby do |env, machine|
+      # The vagrant-libvirt plugin has a race condition getting
+      # virtual bridges and IPs. So, we make sure concurrent processes
+      # are spread out with a lock.
+      lock = File.open(File.join(env.home_path, "test-framework.lock"), "w+")
```

Comment:
Does this mean it's in `~/test-framework.lock`?  Might `/tmp` be a better place for it?

Reply:
No, this means a test-framework.lock in the `~/vagrant.d` folder or wherever the global vagrant home path is set up to be.